src/app/modules/template/template-solution-list/template-solution-list.component.css

```diff
@@ -60,4 +60,8 @@ mat-icon-button:hover {
 mat-paginator {
   background-color: #fafafa;
   border-top: 1px solid #e0e0e0;
+}
+.back-button {
+  color: rgb(0, 116, 182, 1) !important;
```

Comment:
use !important if only if the color is not getting applied when we don't use !important specifier